tuxemon/cli/commands/help.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
+
+from tuxemon.cli.clicommand import CLICommand
+from tuxemon.cli.context import InvokeContext
+from tuxemon.cli.processor import MetaCommand
+
+
+class HelpCommand(CLICommand):
+    """
+    Command to get list available commands and display help for them
+
+    """
+
+    name = "help"
+    description = "Get help"
+    example = "help"
+
+    def invoke(self, ctx: InvokeContext, line: str) -> None:
+        """
+        List available commands and display help for them
+
+        Parameters:
+            ctx: Contains references to parts of the game and CLI interface.
+            line: Input text after the command name.
+
+        """
+        command, tail = ctx.root_command.resolve(ctx, line)
+        if command is not ctx.root_command:
+            print(command.description)
+            print()
+        if command.example:
+            print("Example:")
+            print(f"  > {command.example}")
```

Comment:
Why the arrow? Are you sure examples will be one liners?

Reply:
the arrow is just for decoration.  it looks like a prompt, and you often see that in webpages with a `$` for instance.  the cli doesn't support multiline input, so there would be no example of a command with more than one line